airflow/map.jinja

```diff
@@ -20,6 +20,9 @@ Debian:
   auth:
     engine: password
     enabled: false
+  database:
```

Comment:
I would prefer deepening the structure:
```
database:
  pool:
    size:
    recycle:
```
as it looks better and makes more sense :-)